### PR TITLE
Various UI fixes

### PR DESCRIPTION
**Pull request by amitch23**
## Description

This PR:

* Updates the text on the contact info screen
* Changes the background color of move type selector buttons to a lighter gray
* Removes unimplemented home page links
* Adds link to new window for "Find Weight Scales" on home page if submitted move/ppm.

## Code Review Verification Steps

* [x] All tests pass.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [ ] Any migrations/schema changes also update the diagram in docs/schema/dp3.sqs
* [ ] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [ ] This works in IE.
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @willowbl00
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Request review from a member of a different team.
* [x] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/157194955) 
* [Pivotal story](https://www.pivotaltracker.com/story/show/157575922) 
* [Pivotal story](https://www.pivotaltracker.com/story/show/157666279)

### CODE REVIEW

File: src/scenes/Landing/MoveSummary.jsx [hunk 1]
```diff
@@ -70,8 +70,13 @@ export const MoveSummary = props => {
                 </div>
                 <div className="step-links">
                   <span>
-                    <a>Request Storage</a> | <a>Find Weight Scales</a> |{' '}
-                    <a>Report a Problem</a> | <a>Cancel Shipment</a>
+                    <a
+                      href="https://www.move.mil/resources/locator-maps"
+                      target="_blank"
+                      rel="noopener noreferrer"
```
Comment (jim) [inline src/scenes/Landing/MoveSummary.jsx]: (For my own knowledge) Is this `rel` attribute something we should use every time we're using `target="_blank"`?

Comment (tinyels) [reply src/scenes/Landing/MoveSummary.jsx]: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Comment (jim) [reply src/scenes/Landing/MoveSummary.jsx]: Yikes! TIL.

Comment (amitch23) [reply src/scenes/Landing/MoveSummary.jsx]: it was because of this error: https://circleci.com/gh/transcom/mymove/11170

**Pull request merged**